app/scripts/hexData.js

```diff
@@ -0,0 +1,96 @@
+const __ = {};
+const _L = {
+  iconId: 'shiny-tools-logo'
+};
+const _S = {
+  iconId: 'shiny-semantic',
+  title: 'shiny.semantic',
+  isInteractive: true
+};
+const _D = {
+  iconId: 'semantic-dashboard',
+  title: 'semantic.dashboard',
+  isInteractive: true
+};
+const _I = {
+  iconId: 'shiny-info',
+  title: 'shiny.info',
+  isInteractive: true
+};
+const _R = {
+  iconId: 'shiny-router',
+  title: 'shiny.router',
+  isInteractive: true
+};
+const _N = {
+  iconId: 'shiny-i18n',
+  title: 'shiny.i18n',
+  isInteractive: true
+};
+const _T = {
+  text: '... shiny.tools'
+}
+const _W = {
+  text: 'Welcome to the world of ...'
+}
+
```

Comment:
Here it will be great to have a short comment that these constants are described in README